pkg/eventing/deployment.go

```diff
@@ -74,6 +79,7 @@ func newEventMeshPublisherDeployment(
 		WithBEBEnvVars(GetPublisherDeploymentName(*eventing), publisherConfig, eventing),
 		WithLogEnvVars(publisherConfig, eventing),
 		WithPriorityClassName(PriorityClassName),
+		WithSpecReplicas(replicas),
```

Comment:
why do we need replicas now? HPA will set them for us

Reply:
Removed